Makefile

```diff
@@ -0,0 +1,90 @@
+.DEFAULT_GOAL		:=help
+SHELL				:=/bin/bash
+
+.PHONY: help
+help: ## Show this help
+	@grep -E '^[a-zA-Z\.\-]+:.*?## .*$$' $(MAKEFILE_LIST) | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
+
+.PHONY: dev
+dev:
+	$(eval DEPLOY_ENV=dev)
+	$(eval AZURE_SUBSCRIPTION=s165-teachingqualificationsservice-development)
+
+read-keyvault-config:
+	$(eval KEY_VAULT_NAME=$(shell jq -r '.key_vault_name' terraform/workspace_variables/$(DEPLOY_ENV).tfvars.json))
+	$(eval KEY_VAULT_SECRET_NAME=$(shell jq -r '.key_vault_secret_name' terraform/workspace_variables/$(DEPLOY_ENV).tfvars.json))
+
+read-deployment-config:
+	$(eval SPACE=$(shell jq -r '.paas_space' terraform/workspace_variables/$(DEPLOY_ENV).tfvars.json))
+	$(eval POSTGRES_DATABASE_NAME=$(shell jq -r '.postgres_database_name' terraform/workspace_variables/$(DEPLOY_ENV).tfvars.json))
```

Comment:
postgres_database_name is not provided in parameters file

Reply:
Since we are not using a parameter to store `postgres_database_name` i think below command in Makefile makes more sense.

	`$(eval POSTGRES_DATABASE_NAME="apply-for-qts-in-england-$(DEPLOY_ENV)-pg-svc")`

@saliceti  @merrilmathew